tools/scylla-nodetool.cc

```diff
@@ -1427,10 +1430,17 @@ void repair_operation(scylla_rest_client& client, const bpo::variables_map& vm)
     std::vector<sstring> keyspaces, tables;
     if (vm.contains("keyspace")) {
         auto res = parse_keyspace_and_tables(client, vm, true);
+        auto uses_tablets = keyspace_uses_tablets(client, res.keyspace);
+        if (uses_tablets) {
+            throw std::invalid_argument("nodetool repair repairs only vnodes-based keyspaces");
```

Comment:
s/vnodes-based keyspace/vnode keyspaces/

Please fix other occurences of this too.